crates/rpc/rpc/src/eth/core.rs

```diff
@@ -270,6 +271,9 @@ pub struct EthApiInner<Provider: BlockReader, Pool, Network, EvmConfig> {
 
     /// Guard for getproof calls
     blocking_task_guard: BlockingTaskGuard,
+
+    /// Transaction broadcast channel
+    raw_tx_sender: EventSender<Bytes>,
```

Comment:
I'd prefer to use the actual raw tokio::broadcast::Sender here and not the helper